pkg/apis/ark/v1/backup.go

```diff
@@ -56,6 +58,9 @@ type BackupSpec struct {
 
 	// Hooks represent custom behaviors that should be executed at different phases of the backup.
 	Hooks BackupHooks `json:"hooks"`
+
+	// StorageLocation is a string containing the name of a BackupStorageLocation where the backup should be stored.
```

Comment:
Super nit-picky comment incoming: it's not necessary to describe the type of the variable on the comment, because it's redundant. The comment should be limited to what the variable does or is for. There are existing variables that are described this way as well.

Reply:
I agree it's redundant. In many cases this is fitting in with the existing format (see https://github.com/heptio/ark/blob/master/pkg/apis/ark/v1/backup.go#L223, for example).

I'd like to remove the redundancy in the comments across the code base, but I'd like to be consistent.